src/main/java/connection/DBCPDataSource.java

```diff
@@ -0,0 +1,50 @@
+package connection;
+
+
+import org.apache.commons.dbcp2.BasicDataSource;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Properties;
+
+
+public class DBCPDataSource {
+    private static String JDBC_URL = "jdbc:mysql://localhost:3306/ecommerce_store";
+    private static String USERNAME = "tester";
+    private static String PASSWORD = "Pass123#d";

```

Comment:
I think it'd better to keep sensitive credentials like username, password in properties file and here just connect with them.

Reply:
Yes I agree